sdk/synapse/Azure.Analytics.Synapse.Artifacts/api/Azure.Analytics.Synapse.Artifacts.netstandard2.0.cs

```diff
@@ -5253,6 +5259,13 @@ public partial class PurviewConfiguration
         public PurviewConfiguration() { }
         public string PurviewResourceId { get { throw null; } set { } }
     }
+    public partial class QuickbaseLinkedService : Azure.Analytics.Synapse.Artifacts.Models.LinkedService
+    {
+        public QuickbaseLinkedService(object url, Azure.Analytics.Synapse.Artifacts.Models.SecretBase userToken) { }
+        public object EncryptedCredential { get { throw null; } set { } }
+        public object Url { get { throw null; } set { } }
```

Comment:
Same comment for `EncryptedCredential` and `Url`-- please give these strong types instead of using `object`.  Per the Azure SDK .NET guidelines, we should [use `System.Uri` to represent URIs](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-primitives-uri).